content/2021-12-14-brigade-2.1.0.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Brigade 2.1.0"
+slug: "2021-12-14-brigade-2.1.0"
+description: "Brigade 2.1.0 has been released"
+date: "2021-12-14"
+author: "Kent Rancourt"
+summary: "Brigade 2.1.0 has been released."
+---
+
+Good day, Brigadiers! Today we have _minor_ news for you -- that is we're announcing our first minor release of Brigade 2.
+
+We certainly never imagined Brigade v2.1.0 being released only two weeks after Brigade v2.0.0 -- and we always imagined v2.1.0 being quite feature-packed. Instead, this release is quite _boring_, but we love keeping our community updated, so here we are!
+
+We've been working hard to bring you a GA release of our new [GitHub event gateway](https://github.com/brigadecore/brigade-github-gateway). In the course of that work, we uncovered something we'd not previously anticipated -- that script authors may at times wish to access an event's qualifiers or labels. While that capability isn't strictly required for the GitHub event gateway to function, we found the best examples we could concoct for that gateway did require that capability. This requirement called for changes to the Brigadier library (used by script authors for handling events and launching jobs) and, by extension, the worker image, which also utilizes that library. To unblock our work on the GitHub event gateway, we've released Brigade v2.1.0 with this _one_ new feature.
+
+If you're already rolling with Brigade v2.0.0, there is no compelling reason for you to upgrade at this time unless you have a specific need to access an event's qualifiers or labels from within your scripts.
+
+That's all for today.
+
+Remember to find us on [Slack](https://slack.brigade.sh) if you have questions or need help getting started!
```

Comment:
This link only helps if someone is signed in to k8s Slack on the device where they read this post. Spelling out the Slack team and channel (Kubernetes slack, #brigade) lets people join more easily from the device of their choice.